server/mail/mail.go

```diff
@@ -235,7 +241,13 @@ func dialTimeout(addr string) (client *smtp.Client, err error) {
 		}
 	}()
 
-	conn, err := net.DialTimeout("tcp", addr, 28*time.Second)
+	var conn net.Conn
+	if tlsConfig == nil {
+		conn, err = net.DialTimeout("tcp", addr, 28*time.Second)
+	} else {
+		conn, err = tls.DialWithDialer(&net.Dialer{Timeout: 28 * time.Second}, "tcp", addr, tlsConfig)
```

Comment:
Any idea why `28` is used? Can you make it a const variable? It is used here in 2 places.

Reply:
Best guess is wanting to keep it under the function level timeout at 30s.  There is an additional comment above regarding battle timing attacks with longer timeouts.